packages/webhooks/src/features/event/handler.ts

```diff
@@ -33,7 +34,7 @@ export async function birthRegisteredHandler(
   request: Hapi.Request,
   h: Hapi.ResponseToolkit
 ) {
-  const bundle = request.payload as fhir.Bundle
+  const bundle = request.payload as RegisteredRecord
```

Comment:
For these it would be nice to have some sort of a `validateRecord` etc so we wouldn't need to cast here. Good for now however